addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -133,6 +133,10 @@ class MessageList extends Component {
                 case 'new-messages-loaded':
                     this.threadView.markComponentHintProcessed(hint);
                     break;
+                case 'most-recent':
```

Comment:
hint should be named according to its origin (eg. new pending message?), not according to its goal